### ISSUE TITLE
Failure to load default permission map

### ISSUE THREAD

**Issue by jonagard** [SELinuxProject/setools#9]
### Description

Version:  setools 4.1.1-8.1.fc28
OS: Fedora Core 28

With setools 4.1.1-8.1, I am seeing the following error when I try to run `apol`:

```
Failed to load default permission map: /usr/lib64/python3.6/site-packages/setools/perm_map:391:Invalid class declaration: ['sendto', 'w', '10']
```

Python 3 version:

```
[jonagard@localhost ~]$ python3 --version
Python 3.6.6
```

I do not see this error with 4.1.1-6.

### Reproduction

- On a Fedora Core 28 machine
- `sudo dnf install setools setools-gui`
- `apol`
- Will see the error above in both the GUI and the terminal
- `sudo dnf remove setools setools-gui`
- `sudo dnf install setools-4.1.1-6.fc28.x86_64 setools-gui-4.1.1-6.fc28.x86_64`
- `apol`
- apol opens successfully

**Comment by pebenito** [comment]
Since this is a problem with the Fedora packaging (especially since one version works and another doesn't), you should file a bug in the Fedora bugzilla, as we can't fix it here. It looks like there were errors in the permission map introduced in the SCTP patch.

**Comment by pcmoore** [comment]
FYI @bachradsusi.

**Comment by bachradsusi** [comment]
It's indeed a bug in 0004-Add-support-for-SCTP-protocol.patch. I'm working on the fix. Thanks!

**Comment by bachradsusi** [comment]
It should be fixed in  setools-4.1.1-9.fc28 - https://bodhi.fedoraproject.org/updates/FEDORA-2018-8d9ac6a0c4
